### DISCUSSION TITLE
Slider and Indicator Issue

### DISCUSSION THREAD

**Discussion by Captain-DAB** [primefaces/community#57] [PrimeIcons]
Actually, my carousel is working perfectly but the slider and indicator aren't displaying well. It is bringing dummy color(light) for the two not the original color that is supposed to be there. I thought bootstrap was the issue but I was wrong, I removed the bootstrap links and it did the same thing. I hope for solution as soon as possible please.

**Comment by atkntepe** [comment]
Hi,
To help you, please share the project you are using (PrimeVue, PrimeNG, etc.) and some screenshots that could help us understand the issue better.
